src/Microsoft.ML.Core/Microsoft.ML.Core.csproj

```diff
@@ -12,6 +12,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="System.CodeDom" Version="$(SystemCodeDomPackageVersion)" />
```

Comment:
Is this necessary?  I don't believe it is. The `IndentedTextWriter` should be in the `netstandard` assembly, and no extra packagereference is needed.